snapcraft/tests/commands/test_help.py

```diff
@@ -69,6 +69,15 @@ def test_print_module_help_for_valid_plugin(self):
                          'The help message does not start with {!r} but with '
                          '{!r} instead'.format(expected, output))
 
+    def test_print_module_with_dashes_help_for_valid_plugin(self):
+        main(['help', 'plainbox-provider'])
+
+        expected = ' Create parts containing'
+        output = self.fake_terminal.getvalue()[:len(expected)]
+        self.assertEqual(output, expected,
+                         'The help message does not start with {!r} but with '
+                         '{!r} instead'.format(expected, output))
```

Comment:
Thanks for the feedback, working on it. I have to suggestions that could be worked in the future:

I think testtools startswith-endswith does make the test easier to understand, so it would be nice to also add it to the rest of tests doing similar asserts in test_help.py.

Also I realized that plugins help message is standard (e.g The 'plugin name' plugin is...), but the one that I'm using for this test 'plainbox-provider' isn't, so we should modify it so it prints same message format.